web/src/components/organisms/DeleteModal.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Button, Modal } from "flowbite-react";
+import { HiOutlineExclamationCircle } from "react-icons/hi";
+
+function DeleteModal({ handleDelete, orderId }) {
```

Comment:
1. use arrow function
2. no need to pass orderId, you can get orderId in hooks when calling handleDelete
```suggestion
const = DeleteModal({ handleDelete }) => {
```